### DISCUSSION TITLE
Calibre is not opening some kindle ebooks properly (weird characters)

### DISCUSSION THREAD

**Discussion by JibJib007** [noDRM/DeDRM_tools#507] [Q&A] [answered]
Hello.

I've bought some ebooks from amazon, opened and extracted the files from Bluestacks software, opened them all on Calibre. It was working fine since last year. This week I've tried to convert 10 of new acquired books to PDF, but unfortunately 4 of them are not opening,  neither converting properly, showing strange characters, all messy. I'm using Calibre 7.6. I'm not an expert on these issues, so I'm asking for help.

Thank you
![pdf not working](https://github.com/noDRM/DeDRM_tools/assets/161980453/0fd4bc41-b271-4fd5-af12-08648a11c646)

**Comment by ElleKayEm** [comment, marked as answer]
Yeah, that's still encrypted.  Probably the other ones were DRM-free to begin with.  Extract your key again and use this version of DeDRM: https://github.com/noDRM/DeDRM_tools_autorelease/releases/tag/autorelease_bf2471e65b1f52bb5292caeba70a9aea31bf6653

**Reply by JibJib007** [reply]
Worked fine! Phew! I thought I lost my money for nothing. Many thanks!!
